docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1545,6 +2006,40 @@ Office.SeriesTime.setStartTime:
 
     // Alternative: Set the start time of all instances of a recurring appointment series to 11:30 PM.
     seriesTimeObject.setStartTime("T23:30:00");
+Office.Subject.getAsync:
+  - |-
+    Office.context.mailbox.item.subject.getAsync(callback);
+
+    function callback(asyncResult) {
+        var subject = asyncResult.value;
+    }
+Office.Subject.setAsync:
+  - |-
+    Office.context.mailbox.item.subject.setAsync("New subject!", function (asyncResult) {
+        if (asyncResult.status == "failed") {
+            console.log("Action failed with error: " + asyncResult.error.message);
+        }
+    });
+Office.TaskSuggestion:
+  - |-
+    var item = Office.context.mailbox.item;
+    // Get an array of strings that represent task suggestions in the current item's body.
+    var taskSuggestions = item.getEntitiesByType(Office.MailboxEnums.EntityType.TaskSuggestion);
+    console.log("There are " + taskSuggestions.length + " task suggestions.")
+    taskSuggestions.forEach(function (taskSuggestion) {
+        console.log("Assignees: " + JSON.stringify(taskSuggestion.assignees));
+        console.log("Task: " + JSON.stringify(taskSuggestion.taskString));
+    });
+Office.Time.getAsync:
+  - |-
+    Office.context.mailbox.item.start.getAsync(function (asyncResult) {
+        if (asyncResult.status == "failed") {
```

Comment:
I just noticed these are comparing status to a string literal. Shouldn't they compare to an enum instead?

Reply:
We have examples with each option (enum and string literal)